ldelf/ta_elf_rel.c

```diff
@@ -283,7 +283,8 @@ static void resolve_sym(const char *name, vaddr_t *val, struct ta_elf **mod,
 		if (err_if_not_found)
 			err(res, "Symbol %s not found", name);
 		else
-			*val = 0;
+			if (val)
```

Comment:
I believe the indentation is misleading, we should be better off writing this as:
``` C
else if (val)
        *val = 0;
```

Reply:
Alright